src/hyperdeck.ts

```diff
@@ -18,40 +18,43 @@ class QueuedCommand {
 	public readonly promise: Promise<any>
 	public readonly command: AbstractCommand
 
-	public resolve: (res: any) => void
-	public reject: (res: ErrorResponse) => void
-
-	constructor (command: AbstractCommand) {
+	constructor(command: AbstractCommand) {
 		this.command = command
 		this.promise = new Promise((resolve, reject) => {
 			this.resolve = resolve
 			this.reject = reject
 		})
 	}
+
+	resolve(_res: any) {
+		//
```

Comment:
perhaps these should throw an error? That will help with debugging if in the future they dont get overridden